packages/auth/src/authClients/firebase.ts

```diff
@@ -1,11 +1,15 @@
-import type FirebaseNamespace from 'firebase/app'
+import type { FirebaseApp } from '@firebase/app'
+import type { CustomParameters, OAuthProvider, User } from '@firebase/auth'
+import type FirebaseAuthNamespace from '@firebase/auth'
 
 import { AuthClient } from './'
 
-export type Firebase = typeof FirebaseNamespace
+export type FirebaseAuth = typeof FirebaseAuthNamespace
+export type FirebaseUser = User
 
 // @TODO: Firebase doesn't export a list of providerIds they use
 // But I found them here: https://github.com/firebase/firebase-js-sdk/blob/a5768b0aa7d7ce732279931aa436e988c9f36487/packages/rules-unit-testing/src/api/index.ts
+// NOTE: 2021-09-15 firebase now exports a const/enum ProviderId which could perhabps be used in place of this
```

Comment:
Did you try importing this type by any chance? Would be nice to finally not duplicate this code!